viewer-prototype/src/browser/trace-viewer/components/timegraph-output-component.tsx

```diff
@@ -105,19 +109,36 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
                 timegraphTree: treeResponse.model.entries,
                 styleModel: styleResponse.model
             });
+            this.chartLayer.updateChart();
         }
+
+        if (this.state.collapsedNodes !== _prevState.collapsedNodes && this.chartLayer) {
```

Comment:
Should this be else if instead of if? Also, this.chartLayer again will always be true